diff --git a/src/schema.rs b/src/schema.rs
index defc0d4..f956ec3 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,40 +1,40 @@
 table! {
     projects (id) {
-        id -> Integer,
+        id -> BigInt,
         code -> Text,
         directory -> Nullable<Text>,
         name -> Nullable<Text>,
     }
 }
 
 table! {
     stretches (id) {
-        id -> Integer,
-        subtask_id -> Integer,
-        start -> Integer,
-        end -> Nullable<Integer>,
+        id -> BigInt,
+        subtask_id -> BigInt,
+        start -> BigInt,
+        end -> Nullable<BigInt>,
     }
 }
 
 table! {
     subtasks (id) {
-        id -> Integer,
-        task_id -> Integer,
+        id -> BigInt,
+        task_id -> BigInt,
         branch -> Nullable<Text>,
         description -> Nullable<Text>,
-        active -> Integer,
+        active -> Bool,
     }
 }
 
 table! {
     tasks (id) {
-        id -> Integer,
-        project_id -> Integer,
-        number -> Integer,
-        active_subtask -> Nullable<Integer>,
+        id -> BigInt,
+        project_id -> BigInt,
+        number -> BigInt,
+        active_subtask -> Nullable<BigInt>,
     }
 }
 
 joinable!(stretches -> subtasks (subtask_id));
 joinable!(subtasks -> tasks (task_id));
 joinable!(tasks -> projects (project_id));
